KARL_CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Add `karl-FormActions__button` class and `KarlFormActions` component.
```

Comment:
Ici je pense que ce serait plutot "Add `karl-FormActions__button` class and update `KarlFormActions` component".

Par ailleurs, il faudrait aussi ajouter la modification sur `FormActions` avec le `flex-direction` et le `align-items` dans le `CHANGELOG`.